### PR TITLE
Explain how gossiping works, how to tune and monitor it.

### PR DESCRIPTION
**Pull request by pstibrany**
**What this PR does**: This PR adds basic description of gossip algorithm, and some tuning options.

### CODE REVIEW

File: docs/configuration/arguments.md [hunk 1]
```diff
@@ -188,6 +188,25 @@ prefix these flags with `distributor.ha-tracker.`
 
 Flags for configuring KV store based on memberlist library (works only for the [hash ring](../architecture.md#the-hash-ring), not for the HA Tracker).
```
Comment (pracucci) [inline docs/configuration/arguments.md:189]: I think this should be moved below, before the listing of all flags (right before `- memberlist.nodename`).

**Review by pracucci — APPROVED** [review]
LGTM  (with a nit)

File: docs/configuration/arguments.md (hunk 1 above)
Comment (pstibrany) [reply docs/configuration/arguments.md]: Left the warning on top, moved "Flags ..." closer to flags.

Comment (pracucci) [reply docs/configuration/arguments.md]: Very good, thanks!

**Pull request merged**